pisek/task_jobs/data/manager.py

```diff
@@ -106,6 +98,40 @@ def _get_jobs(self) -> list[Job]:
 
         return jobs
 
+    def _report_unused_inputs(self, unused_inputs: Iterable[InputInfo]) -> None:
+        inputs = list(sorted(unused_inputs, key=lambda inp: inp.name))
+        if self._env.config.checks.no_unused_inputs and len(inputs):
```

Comment:
Empty sequences are falsy:
```suggestion
        if self._env.config.checks.no_unused_inputs and inputs:
```